test/sql/test_metadata.py

```diff
@@ -3902,6 +3903,9 @@ def test_column_info(self):
             c.info["bar"] = "zip"
             assert c.info["bar"] == "zip"
 
+    def test_readonly(self):
+        assert Column(String)._is_readonly is False
```

Comment:
I know my test suite sucks in this regard but I'm trying to not use "assert", so  is_false(Column(String)._is_readonly)  is_true(Column(String, Computed(...))._is_readonly)

is_false and is_true are in sqlalchemy.testing

Reply:
I've used assert since that file uses assert in every test. I'll change it

Reply:
yeah sorry, assert is all over the place and now pytest instruments it even but long term id rather have a function doing this instead of a statement.